ROADMAP.md

```diff
@@ -0,0 +1,185 @@
+# Babel Roadmap
+
+> Not everything is set in stone or has an issue for it! Looking to post these to get more people involved or integrate with other projects.
+
+## Babel 7
+
+Mentioned these in [babel/notes](https://github.com/babel/notes/blob/master/2017/2017-12/dec-21.md#before-the-v7-final) and some of the [high priority items](https://github.com/babel/babel/labels/Priority%3A%20High).
+
+## Ecosystem
+
+### Remake `compat-table` used in preset-env
+
+> https://github.com/kangax/compat-table could use a remake, ideally work with browser vendors on this
+    
+- There is also https://github.com/mdn/browser-compat-data
+- Also use data from test262?
+- Run tests against real browsers
+- have a data-only format
+- Need continued maintainence
+
+### Polyfill behavior
+
+> This is regarding https://github.com/babel/babel/tree/master/packages/babel-preset-env#usebuiltins-usage
+
+- Allow any substitute polyfill instead of `core-js`. You should be able to override anything (custom `Promise`, etc)
+- Make `"usage"` option the default after it is stable.
+
+### Build/publish workflow
+
+- Guide on compiling/publishing ES2015+, .mjs, etc: https://twitter.com/philwalton/status/908082461799616512
```

Comment:
Would this guide include a you might not need section. With Node having 99% ES2015+ it might be nice thing to avoid unneeded build steps. Maybe something in `@babel/env`.